ncbi/src/main/java/org/reactome/release/UniProtReactomeEntry.java

```diff
@@ -0,0 +1,390 @@
+	 * @param graphDBSession Neo4J Driver Session object for querying the graph database
+	 * @return Map of UniProt accession to set of Reactome Events representing top level pathways in Reactome
+	 */
+	public static Map<String, Set<ReactomeEvent>> fetchUniProtAccessionToTopLevelPathways(Session graphDBSession) {
+		if (uniprotAccessionToTopLevelPathwaysCache.containsKey(graphDBSession)) {
+			return uniprotAccessionToTopLevelPathwaysCache.get(graphDBSession);
+		}
+
+		logger.info("Computing UniProt to Top Level Pathways");
+
+		Map<String, Set<ReactomeEvent>> uniprotAccessionToTopLevelPathways =
+			fetchUniProtAccessionToReactomeEvents(graphDBSession).entrySet().stream().collect(Collectors.toMap(
+			Map.Entry::getKey,
+			entry -> entry.getValue()
+					.stream()
+					.filter(reactomeEvent ->
+						PathwayHierarchyUtilities
+							.getTopLevelPathwayIds(graphDBSession)
+							.contains(reactomeEvent.getDbId())
+					)
+					.collect(Collectors.toSet())
+		));
+		uniprotAccessionToTopLevelPathwaysCache.put(graphDBSession, uniprotAccessionToTopLevelPathways);
+
+		logger.info("Finished computing UniProt to Top Level Pathways");
+
+		return uniprotAccessionToTopLevelPathways;
+	}
+
+	/**
+	 * Retrieves, from the graph database, a Map of UniProt accession to the set of events (both Pathways and
+	 * Reaction Like Events in which each UniProt accession participates
+	 * @param graphDBSession Neo4J Driver Session object for querying the graph database
+	 * @return Map of UniProt accession to set of Reactome Events in Reactome
+	 */
+	public static Map<String, Set<ReactomeEvent>> fetchUniProtAccessionToReactomeEvents(Session graphDBSession) {
+		if (uniprotAccessionToReactomeEventCache.containsKey(graphDBSession)) {
+			return uniprotAccessionToReactomeEventCache.get(graphDBSession);
+		}
+
+		logger.info("Computing UniProt to Reactome events");
+
+		Map<Long, ReactomeEvent> eventCache = ReactomeEvent.fetchReactomeEventMap(graphDBSession);
+		Map<String, Set<Long>> uniprotAccessionToReactionLikeEventId = fetchUniProtAccessionToRLEId(graphDBSession);
+		Map<Long, Set<Long>> rleToPathwayId = PathwayHierarchyUtilities.fetchRLEIdToPathwayId(graphDBSession);
+
+		AtomicInteger count = new AtomicInteger(0);
+		Map<String, Set<ReactomeEvent>> uniprotAccessionToReactomeEvent = new ConcurrentHashMap<>();
+		uniprotAccessionToReactionLikeEventId.keySet().parallelStream().forEach(uniprotAccession -> {
+			Set<Long> reactionLikeEventIds =
+				uniprotAccessionToReactionLikeEventId.computeIfAbsent(uniprotAccession, k -> new HashSet<>());
+
+			// Add RLEs containing UniProt accession
+			uniprotAccessionToReactomeEvent
+				.computeIfAbsent(uniprotAccession, k -> new HashSet<>())
+				.addAll(reactionLikeEventIds.stream().map(eventCache::get).collect(Collectors.toSet()));
+
+			for (long reactionLikeEventId : reactionLikeEventIds) {
+				Set<Long> pathwayIds = rleToPathwayId.computeIfAbsent(reactionLikeEventId,
+																			  k -> new HashSet<>());
```

Comment:
The spacing here brings this line far to the right on github

Reply:
Yeah, this was introduced by IntelliJ from line wrapping.  It's fixed now!